src/main/java/L4Z6/BillCalculator.java

```diff
@@ -0,0 +1,27 @@
+package L4Z6;
+
+public class BillCalculator {
+    private double bill;
+    private float service;
+
+    public double calculate(double bill, float service) {
+        this.bill = bill;
+        this.service = service;
```

Comment:
Po co to przypisanie do pól? 
Wystarczy: `      return bill + service;`
I analogicznie wszędzie indziej